schema.sql

```diff
@@ -8,38 +8,40 @@ CREATE SCHEMA web3storage
     github TEXT,
     public_address TEXT NOT NULL,
     used_storage bigint DEFAULT 0,
-    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL
+    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+    updated_at timestamp with time zone
   )
   CREATE TABLE auth_tokens (
     id BIGSERIAL PRIMARY KEY,
     name TEXT NOT NULL,
     secret TEXT NOT NULL,
     user_id bigint NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
-    deleted_at timestamp with time zone DEFAULT timezone('utc'::text, now())
+    deleted_at timestamp with time zone
   )
   CREATE TABLE uploads (
     id BIGSERIAL PRIMARY KEY,
     user_id bigint NOT NULL,
-    auth_token_id bigint NOT NULL,
+    auth_token_id bigint,
```

Comment:
Uploads made via the website use Magic.link tokens not our API tokens and so these uploads have a `null` auth token.